contracts/hydro/src/msg.rs

```diff
@@ -149,3 +214,8 @@ pub struct LiquidityDeployment {
     // if this is a "repeating" proposal
     pub remaining_rounds: u64,
 }
+
+#[derive(Serialize, Deserialize)]
+pub enum ReplyPayload {
```

Comment:
I added a new `ReplyPayload` structure because I didn't want to mix it up with the existing one that is specific to the ICQs and will be moved into a different SC at some point.